content/millennium/r4/general-clinical/allergy-intolerance.md

```diff
@@ -0,0 +1,111 @@
+---
+title: AllergyIntolerance | R4 API
+---
+
+# AllergyIntolerance
+
+* TOC
+{:toc}
+
+## Overview
+
+The AllergyIntolerance resource provides the clinical assessment of a patient's allergy or intolerance when exposed to a specific substance or class of substance including information about the adverse reaction.  Substances include, but are not limited to, medications, foods, environment (such as plants and animals), and insect bites.  The Allergy/Intolerance list exists as a patient safety tool for clinical decision support when ordering medications and nutrition or guiding clinical treatments.  This resource does NOT include adverse reactions or adverse events which are expected for the circumstance such as an over-dose or drug-drug interaction or an error/failure in the clinical process.  References to implicitRules and modifierExtensions are NOT supported and will fail a Create or Update request.
+
+No Known Allergies (NKA) or No Known Medication Allergies (NKMA) will be conveyed with predefined codes while Not Asked is conveyed via the absence of information (empty query response).  Consumers can supply the negation codes No Known Allergies (160244002) or No Known Medication Allergies (409137002) as long as there are no other active allergies on the patient's profile.  If there are other active allergies on the patient's profile and the consumer tries to add one of the codes above, the service will throw an exception.  
+
+If NKA and/or NKMA exist on the patient's allergy profile and a new allergy is added, the NKA and NKMA will be canceled.  
+
+If the consumer is sending in what is deemed to be a duplicate (codified or free text), the service will update the existing allergy or reaction rather than adding a duplicate allergy or reaction.
+
+Assuming a patient safety or decision support use case, a consumer should avoid querying by status since it is not a required field and can ignore any entered-in-error allergies or intolerances.
+
+The following fields are returned if valued:
+
+* [Allergy id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Allergy code](http://hl7.org/fhir/DSTU2/allergyintolerance-definitions.html#AllergyIntolerance.code){:target="_blank"}
```

Comment:
nit: Shouldn't this url be - http://hl7.org/fhir/R4/allergyintolerance-definitions.html#AllergyIntolerance.code ?

Reply:
Yes: https://github.com/cerner/fhir.cerner.com/pull/259/commits/68a12f46d40a8a5e11fb741d89231148d82c870f